ams/server/src/main/java/com/netease/arctic/server/dashboard/ServerTableDescriptor.java

```diff
@@ -52,13 +50,8 @@ public class ServerTableDescriptor extends PersistentBase {
   public ServerTableDescriptor(TableService tableService, Configurations serviceConfig) {
     this.tableService = tableService;
 
-    ExecutorService executorService =
-        Executors.newFixedThreadPool(
-            serviceConfig.getInteger(ArcticManagementConf.TABLE_MANIFEST_IO_THREAD_COUNT),
-            new ThreadFactoryBuilder()
-                .setDaemon(true)
-                .setNameFormat("table-manifest-io-%d")
-                .build());
+    // All table formats will jointly reuse the work thread pool named iceberg-worker-pool-%d
```

Comment:
The dashboard requires a separate thread pool to prevent user queries from being blocked by background plan tasks.